include/ioq.hrl

```diff
@@ -19,6 +19,8 @@
 -define(DISPATCH_SINGLE_SERVER, "single_server").
 -define(DISPATCH_SERVER_PER_SCHEDULER, "server_per_scheduler").
 
+-define(IOQ2_SEARCH_SERVER, ioq_server_search).
```

Comment:
Well, it's deliberate that the server itself is `ioq_server_search` so that it starts with `ioq_server_` to utilize this logic flagging it as an IOQ2 pid: https://github.com/apache/couchdb-ioq/blob/d400e34e3637289081bab125b9c39e3c7b8a9d10/src/ioq_sup.erl#L87-L88

I thought the define name was more natural, but we could reverse the name so they're identical.

Reply:
Nah, I don't think it's a big deal, just wanted to check.